hooks/taskcreator.js

```diff
@@ -66,10 +66,17 @@ exports.TaskCreator = TaskCreator;
 class MockTaskCreator extends TaskCreator {
   constructor() {
     super({credentials: {}});
+    this.shouldFail = false;
     this.fireCalls = [];
   }
 
   async fire(hook, payload, options) {
+    if (this.shouldFail) {
+      let err = new Error("uhoh");
```

Comment:
I make a habit of using this for errors in testing, since it's clear (or at least easy to grep for) when debugging the tests that this is an intentional error.